source/includes/_creatives.md

```diff
@@ -0,0 +1,156 @@
+## Creatives
+
+### Resource Overview
+
+| Method | URI Format |
+|---|---|
+| GET | /client_reports/creatives/[gmaid]?[query_params] |
+#### API Name: creatives
+### Usage
+Use GET to retrieve information for the Creatives API that will retrieve information on the creative.
```

Comment:
This wording feels redundant.  Could also mention that it's for a specific advertiser.